airflow/www/views.py

```diff
@@ -2233,18 +2234,22 @@ def _clear_dag_tis(
         session: Session,
     ):
         if confirmed:
-            count = dag.clear(
+            tis = dag.clear(
                 start_date=start_date,
                 end_date=end_date,
                 task_ids=task_ids,
                 include_subdags=recursive,
                 include_parentdag=recursive,
                 only_failed=only_failed,
                 session=session,
+                return_cleared_task_instances=True,
             )
-
-            msg = f"{count} task instances have been cleared"
-            return redirect_or_json(origin, msg)
+            log = TaskContextLogger("webserver", call_site_logger=logger)
+            if isinstance(tis, list):  # necessary for mypy
+                for ti in tis:
+                    log.info("Task was manually cleared from the UI, rerunning", ti=ti)
```

Comment:
Nice to have this logging. Can you help me with following questions for my understanding? 
Does this message is appended to S3 task logs? How does this task logging works from web server and scheduler side?

Reply:
There were issues in the first approach, I'm reworking it. I expect it to work like every other task context logging